mig-frankfurt/dataelementhub.rest
# #20 - Add import functionality

**What's in the PR**
*  Add endpoint to enable importing elements to a staging area.
* Add endpoint to retrieve all imports.
* Add endpoint to get import status by Id.
* Add endpoint to get all import members.
* Add endpoint to get an imported element by id.
* Add endpoint to get all members of a stagedElement by id.
* Add endpoint to enable converting the stagedElements to drafts.
* Add endpoint to delete an import by id.


**How to test manually**
* Check if described functionality in openApi documentation (Section "Import and stagedElement Management") is working properly..
